### DISCUSSION TITLE
为什么我的CPU占用率极低

### DISCUSSION THREAD

**Discussion by Cutezsz** [share121/inter-knot#2066] [其他]
打游戏的时候也是这样
![image](https://github.com/user-attachments/assets/c248ac97-70b2-44ea-96db-e53d1bd95b33)

**Comment by 1luik** [comment]
你什么配置啊，这个跟游戏有关系的有游戏的只能一个核或者用gpu加速（a卡只能linux）用

**Reply by 1luik** [reply]
当然也有可能是你软件出bug了用cmd看看功耗

**Comment by Cutezsz** [comment]
![image](https://github.com/user-attachments/assets/e105fc1f-892c-4971-8132-54d0d608f4f9)
![image](https://github.com/user-attachments/assets/10626832-1920-47ef-b763-c4eff68043ed)

系统是win11的，是游戏本，显卡是N卡

**Reply by Cutezsz** [reply]
可能还真不是游戏问题，因为我玩好多游戏都是这样

**Reply by Cutezsz** [reply]
电脑小白，不是特别懂，求指教

**Comment by Regionnor** [comment]
游戏本+4060建议开一下独显直连吧，核显就没必要用了
![image](https://github.com/user-attachments/assets/40e60a6a-1a57-4d44-b794-c5691eae5b5f)
